src/components/home/HomeFooter.jsx

```diff
@@ -0,0 +1,76 @@
+import styled from 'styled-components';
+import { icTravel, icWish, icSearch, icMassage, icProfile } from 'assets';
+import { Link } from 'react-router-dom';
+
+function NavBar() {
+  return (
+    <StyledFooter>
+      <StyledNavToggle>
+        <img src={icSearch} />
+        <span>둘러보기</span>
+      </StyledNavToggle>
+      <Link to={'./Wish'}>
+        <StyledNavToggle>
+          <img src={icWish} />
+          <span>위시리스트</span>
+        </StyledNavToggle>
+      </Link>
+      <Link to={'./'}>
+        <StyledNavToggle>
+          <img src={icTravel} />
+          <span>여행</span>
+        </StyledNavToggle>
+      </Link>
+      <StyledNavToggle>
+        <img src={icMassage} />
+        <span>메세지</span>
+      </StyledNavToggle>
+      <StyledNavToggle>
+        <img src={icProfile} alt="" />
+        <span>프로필</span>
+      </StyledNavToggle>
+    </StyledFooter>
+  );
+}
+
+export default NavBar;
+
+const StyledFooter = styled.footer`
+  position: sticky;
+  bottom: 0;
+  display: flex;
+  width: 100%;
+  padding: 0.9rem 2.15rem 1rem 2.15rem;
+  align-items: center;
+  justify-content: space-between;
+
+  height: 5.8rem;
+
+  background: ${(props) => props.theme.colors.airWhite};
+  border: 0.1rem solid ${(props) => props.theme.colors.airGray2};
```

Comment:
```suggestion
  border-top: 0.1rem solid ${(props) => props.theme.colors.airGray2};
```
border-top만 해줘도 됩니다 !! ☺️